include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -220,7 +220,11 @@ class FEEvaluationBase
    * sums them into the vector @p dst. The function also applies constraints
    * during the write operation. The functionality is hence similar to the
    * function AffineConstraints::distribute_local_to_global. If vectorization
-   * is enabled, the DoF values for several cells are used.
+   * is enabled, the DoF values for several cells are used. The mask can be
+   * used to suppress the write access for some of the vectorized cells, e.g.
```

Comment:
some of the vectorized cells -> some of the cells contained in the current cell vectorization batch